processor/src/main/java/com/linecorp/decaton/processor/metrics/Metrics.java

```diff
@@ -44,6 +45,28 @@ private Metrics(AvailableTags availableTags) {
         this.availableTags = availableTags;
     }
 
+    public class SubscriptionMetrics {
+        private Timer processDuration(String scope) {
+            return Timer.builder("subscription.process.durations")
+                        .description(String.format(
+                                "Time spent for processing %s in consuming loop", scope))
+                        .tags(availableTags.subscriptionScope().and("scope", scope))
```

Comment:
- `scope` appears twice as different meaning here (subscription "scope" and consuming loop scope), so `part` sounds more appropriate

Reply:
That's reasonable, but may I choose `section` instead? It feels more natural to me.